build_deploy.sh

```diff
@@ -15,6 +15,11 @@ readonly QUAY_TOKEN="${QUAY_TOKEN:-}"
 readonly RH_REGISTRY_USER="${RH_REGISTRY_USER:-}"
 readonly RH_REGISTRY_TOKEN="${RH_REGISTRY_TOKEN:-}"
 
+if [[ "${GIT_BRANCH}" == 'stable' ]]; then
+    IMAGE_ALIAS='qa'
+else
+    IMAGE_ALIAS='latest'
```

Comment:
@cutwater I assume this will also tag the image with the git sha as it does today?

Reply:
Yes, exactly